examples/hubert/train.py

```diff
@@ -0,0 +1,166 @@
+import pathlib
```

Comment:
Since this is a script, can you add shebang line and docsrtring, the reuse the docsrting for help message of CLI?

```python
#!/usr/bin/env python3
"""docsrting"""
```

then

```
class _Formatter(ArgumentDefaultsHelpFormatter, RawDescriptionHelpFormatter):
    # https://stackoverflow.com/a/18462760
    pass

parser = ArgumentParser(
    description=__doc__,
    formatter_class=_Formatter,
)
```

Reply:
Sure, will update it.